client/lib/push-notifications/service-worker.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *  This file is served as-is as /service-worker.js
+ *  @TODO: Change this :)
```

Comment:
I don't think we'll be able to have more than one service-worker that operates on the same scope. From the [W3C Draft](https://www.w3.org/TR/service-workers/#service-worker-registration-concept):
> The user agents may enable many service worker registrations at a single origin so long as the scope url of the service worker registration differs. A service worker registration of an identical scope url when one already exists in the user agent causes the existing service worker registration to be replaced.

That means that for any service-worker registration that we'll be able to access throughout Calypso, we won't be able to use multiple service-worker files. All the code will need to be combined into a single file. For this reason, I don't think renaming the file to `push-notifications.js` makes sense, unless we figure out a way to build various components into a single `service-worker.js`